### ISSUE TITLE
[BUG] Unable to run kubernetes behind a proxy (https -> http)

### ISSUE THREAD

**Issue by ybizeul** [Leantime/leantime#2043]
*If you have problems with the installation please use our [community forum](https://community.leantime.io)*

**What set up are you using**
* [ ] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**
In absence of `appUrl`, leantime falls back to `getSchemeAndHttpHost()` which returns `http://site.url/` even though a reverse proxy actually serves over https, but addressing the backend container (leantime) over http.

The only option that fixes the issue is `LEAN_APP_URL` but the helm chart doesn't provide a value for this.

**Comment by marcelfolaron** [comment]
You are correct, we'll need to update the helmchart deployment yaml (https://github.com/Leantime/leantime/blob/master/helm/templates/deployment.yaml) to include the latest variables from our environment files https://github.com/Leantime/leantime/blob/master/config/sample.env
